webapp/src/client.ts

```diff
@@ -692,6 +693,14 @@ export const saveChannelAction = async (action: ChannelAction): Promise<string>
     return action.id;
 };
 
+export const updateRunActions = async (playbookRunID: string, actions: RunActions) => {
+    try {
+        return await doPut(`${apiUrl}/runs/${playbookRunID}/actions`, JSON.stringify(actions));
```

Comment:
type here helps even if it's void
```suggestion
        return await doPut<void>(`${apiUrl}/runs/${playbookRunID}/actions`, JSON.stringify(actions));
```

Reply:
I assumed it is void, please use the right type if it returns something

Reply:
Good suggestion, thanks!